js/cloze.js

```diff
@@ -39,8 +39,17 @@
       if (behaviour.caseSensitive !== true) {
         answered = answered.toLowerCase();
       }
-
       for (var i = 0; i < answers.length; i++) {
+        // Damerau-Levenshtein comparison
+        if (behaviour.acceptSpellingErrors === true) {
```

Comment:
I meant adding the new parameter that you introduced to JSDoc, "acceptSpellingErrors", for instance like:
`@param {boolean} behaviour.acceptSpellingErrors Makes answers accept spelling errors. The amount of errors accepted is decided by how long the answer is.`